core/services/ocr2/plugins/functions/types.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 option go_package = "core/services/ocr2/plugins/functions";
 
-package types;
+package functions_types;
```

Comment:
As long as .pb.go resides in `functions` folder, the package name must be `functions`.